include/ecpool.hrl

```diff
@@ -1,3 +1,14 @@
--type callback() :: fun((any()) -> any()).
--type action() :: fun((pid()) -> any()).
--type exec_mode() :: relay | relay_async | {relay_async, callback()} | direct.
\ No newline at end of file
+-type callback() :: mfa() | fun((any()) -> any()).
```

Comment:
We can support a `{fun(), [Arg :: term()]}`.
The worker should `erlang:apply(Fun, Args)`,
and the caller can pass in `{fun mymodule:myfunction/2, [Arg1, Arg2]}`,
which is safer than `fun((any()) -> any())`.

Reply:
The external fun `fun Mod:Func/Arity` is nice but I think It is almost working the same as the form `{M, F, A}`.
I'd like to use `{M, F, A}` here just because {Fun, [Arg]} cannot force the caller pass an *external* fun, it may pass a *local* fun of form `fun() -> foo end` instead.

Reply:
ok.